firebase-dev.json

```diff
@@ -1,10 +1,23 @@
 {
-  "hosting": {
-    "public": "dist",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ]
-  }
+  "hosting":
+    {
+      "site": "yfthig-dev",
+      "public": "dist",
+      "ignore": ["**/devops/**", "**/.*", "**/node_modules/**"],
+      "headers" : [ {
+        "source": "**",
+        "headers" : [
+          { "key" : "Access-Control-Allow-Origin", "value" : "*" },
```

Comment:
Would be nice to locate these settings on one file, that way it's consistent across environments? Not sure if that's easily possible though

Reply:
currently it deploys to differently configured projects on the firebase side (unfortunate).
we'll be migrating those other projects out so that the firebase setup for dev,test,sandbox,prod are all the same.